src/reference/90-Developers-Guide/09-Launcher/01-Launcher-Getting-Started.md

```diff
@@ -154,7 +163,9 @@ class Main extends xsbti.AppMain
 Next, define a configuration file for the launcher. For the above class,
 it might look like:
 
-Then, `publishLocal` or `+publishLocal` the application to make it
+FIXME
```

Comment:
Ah yeah. I think this was machine-converted.

And yeah while we're at it we should move away from org.scala-sbt/xsbt-test/0.12.0/xsbt.test.Main, and use a more acme/foo type example. Good call.